integrations/dbt_bq/README.md

```diff
@@ -0,0 +1,76 @@
+## <span style='color:#ff5f27'> 👨🏻‍🏫 DBT Tutorial with BigQuery </span>
+
```

Comment:
This tutorial shows you how to perform feature engineering in DBT on BigQuery, storing offline computed features in a table in BigQuery (that is mounted as an external feature group in Hopsworks) and online features in Hopsworks. The online features are written to Hopsworks using a Python module that is run on a DataProc cluster. The feature group created in Hopsworks has its offline data stored in BigQuery and its online data stored in Hopsworks online store (RonDB).